stdlib/Test/test/runtests.jl

```diff
@@ -764,10 +764,9 @@ end
         @test_throws InterruptException throw(InterruptException())
     end
 
-    f = tempname()
-
-    write(f,
-    """
+    mktemp() do f, _
+        write(f,
+        """
     using Test
```

Comment:
Can you indent these lines to match the surrounding `"""`s? Otherwise it looks pretty weird. Aside from that, this looks good to me.

Reply:
sure, fixed